src/modules/appeals-service/AppealsService.ts

```diff
@@ -43,7 +43,7 @@ export class AppealsService {
             .then((response: AxiosResponse<string>) => {
                 if (response.status === CREATED && response.headers.location) {
                     loggerInstance()
-                        .info(`${AppealsService.name} - save: created resource ${appeal.id} - ${response.headers.location}`);
+                        .info(`${AppealsService.name} - save: created resource ${response.data} - ${response.headers.location}`);
```

Comment:
should this be r`esponse.data.toString()`?

Reply:
In this case it doesn't really matter because the log statement will always convert this to a string

Reply:
In the return statement it matters because we want to make sure that id that gets returned is a string and not a number for example